### ISSUE TITLE
No predictions appearing

### ISSUE THREAD

**Issue by ywfoo** [janclemenslab/das#26]
Hello! I have tried annotating a file and training a model with these annotations but when I run predictions on the same file, no predictions show up. There is also no error code for it. 

I'd appreciate any help you can offer!

**Comment by postpop** [comment]
Hi, thanks for reaching out! Could be many things. Do you mind sharing the file with the annotations and the model? Maybe I can spot the issue.

**Comment by postpop** [comment]
I can confirm that the network you sent does not produce predictions. But when I take your data, make a data set and train - all with the default parameters - the resulting model does work quite well. Maybe you're on an old version of DAS. We constantly update and fix bugs. Can you re-create the environment and try training again?

**Comment by ywfoo** [comment]
Hello. It seems like the older version (0.24.1) works better than the newer version (0.25.5) for predictions. And another observation I had was that there was no improvement after the first epoch when training with the new version of DAS. Any thoughts on that? Thanks again!

**Comment by postpop** [comment]
With very small training data sets, the performance may converge pretty within one epoch. But this may also be caused by a too high learning rate - you can lower that in the train-dialog. Alternatively/additionally, you can turn on automatic reduction of the learning rate via the "Reduce learning rate patience" - this will reduce the learning rate if the validation loss did not decrease within the specific number of epochs.
